packages/mds-compliance-engine/engine/count_processors.ts

```diff
@@ -86,6 +86,7 @@ export function processCountPolicy(
             delete devicesToCheck[device.device_id]
             delete overflowedVehicles[device.device_id]
           } else if (overflowedVehicles[device.device_id]) {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
The other option is to allow orange-warning-scwiggles everywhere, include GHActions splattering "warning shame" over every build. 

Personally, I can't manage "okay warnings" and "not okay warnings", because that feels like a "dirty output" anti-pattern, where you start ignoring warnings you know are safe, and never see any _useful_ warnings because you've gotten used to filtering them out.

Warnings should be line-disabled when the rule needs to be broken (like here, it's important).